server/views/pages/contacts/manage/listContacts.njk

```diff
@@ -92,22 +103,22 @@
     </table>
 {% endmacro %}
 
-{% macro prisonerDoesNotHaveAnyContactsMessage(journey, contactListType) %}
-    <p>{{ journey.prisoner.firstName | convertToTitleCase }} {{ journey.prisoner.lastName | convertToTitleCase }} does not have any {{ contactListType }} contacts</p>
+{% macro prisonerDoesNotHaveAnyContactsMessage(prisonerDetails, contactListType) %}
+    <p>{{ prisonerDetails.firstName | convertToTitleCase }} {{ prisonerDetails.lastName | convertToTitleCase }} does not have any {{ contactListType }} contacts</p>
 {% endmacro %}
 
 <span class="govuk-caption-l">Manage Contacts</span>
-<h1 class="govuk-heading-l">Contacts for {{ journey.prisoner | formatName(excludeMiddleNames = true) }}</h1>
+<h1 class="govuk-heading-l">Contacts for {{ prisonerDetails | formatName(excludeMiddleNames = true) }}</h1>
 
     {{ govukButton({
         text: "Add prisoner contact",
-        href: "/prisoner/" + journey.prisoner.prisonerNumber + "/contacts/search/" + journey.id,
+        href: "/prisoner/" + prisonerNumber + "/contacts/search/" + (journey.id if journey),
```

Comment:
Known issue that this link doesn't actually work if you drop straight into the landing page.  Next change will be to move contact search into the create journey and this URL will be changing anyway